content/en/tracing/trace_collection/automatic_instrumentation/otel/java.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Using OpenTelemetry Instrumentations with Java
+kind: documentation
+code_lang: java
+type: multi-code-lang
+code_lang_weight: 0
+further_reading:
+    - link: "/opentelemetry/interoperability/"
+      tag: "Documentation"
+      text: "Interoperability with Datadog"
+---
+## Overview
+
+Datadog SDKs support adding instrumentations from OpenTelemetry alongside their built-in instrumentations.
+
+## Compatibility requirements
+
+Support for adding OpenTelemetry instrumentations at runtime was introduced in v1.35.0 of Datadog's SDK for Java.
+
+Each instrumentation should be packaged as an OpenTelemetry [extension](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/examples/extension/README.md).
+The SDK also accepts selected individual instrumentation jars produced by OpenTelemetry's [opentelemetry-java-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main) build,
+for example the [CFX instrumentation jar](https://search.maven.org/search?q=a:opentelemetry-javaagent-jaxws-2.0-cxf-3.0).
+
+**Note**: Use of OpenTelemetry incubator APIs is not currently supported.
+
+## Getting started
+
+To begin using an OpenTelemetry instrumentation with the Datadog SDK for Java:
+1. Set the `dd.trace.otel.enabled` system property or the `DD_TRACE_OTEL_ENABLED` environment variable to true
+2. Download (or build) the OpenTelemetry instrumentation as an extension jar
```

Comment:
What steps are needed to do this?

Reply:
I've reworked the earlier text to refer to OpenTelemetry's example build as an example